content/blog/2021-02-17-cml-runner-prerelease.md

```diff
@@ -0,0 +1,177 @@
+---
+title:
+  'CML Pre-Release Notes: Automatically Train Models in the Cloud with CML 0.3.0'
+date: 2021-02-22
+description: |
+  New features are here to make launching cloud compute for continuous
+  integration workflows shorter, sweeter and easier than ever. Plus, a new
+  GitHub Action to setup CML means more ways to use CML without our Docker
+  container.
+descriptionLong: |
+  New features are here to make launching cloud compute for continuous
+  integration workflows shorter, sweeter and easier than ever. Plus, a new
+  GitHub Action to setup CML means more ways to use CML without our Docker
+  container.
+picture: 2021-02-17/cover.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/cml-0-3-0-pre-release/685
+tags:
+  - CML
+  - Continuous Machine Learning
+  - GitHub Actions
+  - GitLab
+  - Cloud
```

Comment:
💅 sounds goo generic. have we used this tag anywhere before?

Reply:
nah, i'm switching to Terraform